src/main/webapp/feed.html

```diff
@@ -61,10 +64,65 @@
     </script>
 </head>
 <body onload="buildUI()">
-<div id="content">
-    <h1>Message Feed</h1>
-    <hr/>
-    <div id="message-container">Loading...</div>
+<div id="app">
+    <v-app>
+        <v-content>
+            <v-toolbar light class="elevation-0">
+                <v-toolbar-side-icon @click="toggleDrawer" light
+                ></v-toolbar-side-icon>
+
+                <v-toolbar-title>Message Feed</v-toolbar-title>
+
+                <v-spacer></v-spacer>
+
+                <v-btn icon>
+                    <v-icon>search</v-icon>
+                </v-btn>
+
+                <v-btn icon>
+                    <v-icon>apps</v-icon>
+                </v-btn>
+
+            </v-toolbar>
+            <v-navigation-drawer permanent hide-overlay v-show="drawer.open">
+                <v-toolbar flat>
+                    <v-list>
+                        <v-list-tile>
+                            <v-list-tile-title class="title">
+                                Application
+                            </v-list-tile-title>
+                        </v-list-tile>
+                    </v-list>
+                </v-toolbar>
+
+                <v-divider></v-divider>
+
+                <v-list id="navigation" dense class="pt-0">
+                    <v-list-tile
+                            v-for="item in items"
+                            :key="item.title"
+                            :href="item.url"
+                    >
+                        <v-list-tile-action>
+                            <v-icon>{{ item.icon }}</v-icon>
+                        </v-list-tile-action>
+
+                        <v-list-tile-content>
+                            <v-list-tile-title>{{ item.title }}</v-list-tile-title>
+                        </v-list-tile-content>
+                    </v-list-tile>
+                </v-list>
+            </v-navigation-drawer>
+
+            <div id="content">
+
+                <div id="message-container">Loading...</div>
+            </div>
+        </v-content>
+    </v-app>
 </div>
+<script src="https://cdn.jsdelivr.net/npm/vue/dist/vue.js"></script>
+<script src="https://cdn.jsdelivr.net/npm/vuetify/dist/vuetify.js"></script>
+<script type="text/javascript" src="/js/navigationbar-loader.js"></script>>
```

Comment:
I have already added here.

Reply:
No that is different, I mean navigation-loader instead of navigationbar-loader.

Reply:
Oh okay.
I have added navigation-loader on feed.html page.